07-modeling.Rmd

```diff
@@ -405,7 +405,7 @@ These results can then be used in a variety of ways, including examining residua
 #| fig.alt: Residual scatter plot with a x-axis of 'Fitted value of electricity cost' ranging between approximately $0 and $4,000 and a y-axis with the 'Residual of model' ranging from approximatley -$3,000 to $5,000. The points create a slight megaphone shape with largest residuals in the middle of the x-range. A red line is drawn horizontally at y=0.
 fitstats %>%
   ggplot(aes(x = .fitted, .resid)) +
-  geom_point() +
+  geom_point() + 
```

Comment:
Should we add `alpha` here to make it easier to see all the points? Or leave as is?

Reply:
Great idea - with some fiddling, I am using alpha=0.1